src/target/llvm/codegen_llvm.cc

```diff
@@ -88,7 +88,11 @@ void CodeGenLLVM::InitTarget(llvm::TargetMachine* tm) {
       native_vector_bits_ = 128;
     } else {
       native_vector_bits_ = 128;
+#if TVM_LLVM_VERSION >= 100
+      std::string arch_name = std::string(tm->getTargetTriple().getArchName());
```

Comment:
Seems it is always safe to directly do explicit conversion, so I think we can remove the macro guard and use this impl

Reply:
I agree. We could only use `std::string arch_name = std::string(tm->getTargetTriple().getArchName());`

Reply:
Done